### DISCUSSION TITLE
I made a Shell Extension

### DISCUSSION THREAD

**Discussion by WDGASTERtim** [TurboWarp/extensions#363] [Show and tell]
this extension allows you to use the system's internal terminal via scratch using the python server with it you can do everything with your computer run applications create folders on your computer here is the link to the repo: https://github.com/WDGASTERtim/Turbowarp-Shell

**Comment by WDGASTERtim** [comment]
I tested only on MacOs if someone tests on windows and Linux write how it works

**Reply by penta-quark-neutro** [reply]
yo lo he probado en windows y parese funcionar correctamente.

**Reply by oliik2013** [reply]
On windows working

**Comment by GarboMuffin** [comment]
FYI while that server is running, any website you visit could, theoretically, fully compromise your computer. hence we will not be adding something like it due to unfixable security horrors.

**Reply by WDGASTERtim** [reply]
i updated the code

**Comment by Orionband** [comment]
This is rly cool
